[2024-07-16 02:46:09,122] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2021-08-19T00:00:00+00:00 [queued]>
[2024-07-16 02:46:09,197] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: our_first_dag_v2.second_task 2021-08-19T00:00:00+00:00 [queued]>
[2024-07-16 02:46:09,200] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-07-16 02:46:09,202] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2024-07-16 02:46:09,204] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-07-16 02:46:09,281] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): second_task> on 2021-08-19T00:00:00+00:00
[2024-07-16 02:46:09,298] {standard_task_runner.py:52} INFO - Started process 3201 to run task
[2024-07-16 02:46:09,316] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', '2021-08-19T00:00:00+00:00', '--job-id', '866', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp6dcz5dpp', '--error-file', '/tmp/tmphxnp6mi3']
[2024-07-16 02:46:09,329] {standard_task_runner.py:77} INFO - Job 866: Subtask second_task
[2024-07-16 02:46:09,534] {logging_mixin.py:104} INFO - Running <TaskInstance: our_first_dag_v2.second_task 2021-08-19T00:00:00+00:00 [running]> on host f3060829e7b4
[2024-07-16 02:46:09,761] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=coder2j
AIRFLOW_CTX_DAG_ID=our_first_dag_v2
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-19T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-19T00:00:00+00:00
[2024-07-16 02:46:09,768] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2024-07-16 02:46:09,793] {bash.py:158} INFO - Running command: echo hello world, this is the second task, it will run after first task
[2024-07-16 02:46:09,820] {bash.py:169} INFO - Output:
[2024-07-16 02:46:09,833] {bash.py:173} INFO - hello world, this is the second task, it will run after first task
[2024-07-16 02:46:09,835] {bash.py:177} INFO - Command exited with return code 0
[2024-07-16 02:46:09,969] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, execution_date=20210819T000000, start_date=20240716T024609, end_date=20240716T024609
[2024-07-16 02:46:10,276] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-16 02:46:10,302] {local_task_job.py:146} INFO - Task exited with return code 0
